### ISSUE TITLE
Allow for non-rectangular crops

### ISSUE THREAD

**Issue by legovini** [scoute-dich/PDFCreator#4]
It would be very useful to be able to crop by selecting the four corners of a parallelogram, like CamScanner does.

**Comment by scoute-dich** [comment]
I agree, but I found no free image cropping library with such possibilities. So I think it's not possible for me to implement this feature.

**Comment by legovini** [comment]
OpenNoteScanner [1] does it with OpenCV [2], as far as I know.

[1] https://github.com/ctodobom/OpenNoteScanner
[2] https://github.com/ctodobom/OpenCV-3.1.0-Android
